tests/test_config.py

```diff
@@ -91,64 +94,55 @@ def test_Config___init___error(config_file: str, exception: type[Exception], err
     assert str(e.value) == error
 
 
-def test_Config__infer_config_from_basedir(local_db: Path):
-    cfg = testmod.Config(try_infer_environment=False)
-    res = cfg._infer_config_from_basedir(local_db)
-    assert res[1] == "local-db"
-
-
-def test_Config__infer_config_from_basedir_error():
-    cfg = testmod.Config(try_infer_environment=False)
-    with pytest.raises(FileNotFoundError):
-        cfg._infer_config_from_basedir("/blaaa")
-
-
 def test_Config_has_access_lustre():
     cfg = testmod.Config(try_infer_environment=False)
     assert not cfg.has_access_lustre
 
 
-def test_Config_has_access_users_database():
+def test_user_specific_paths_ini():
+    # test if user specific paths.ini file is read
+    CHANGE_NAME = "NAME_CHANGED_FOR_TESTING"
+    CHECK_NAME = "GAWTADSUBSETAASETAL"
+    user_file = os.path.join(const.my_pyaerocom_dir, const.PATHS_INI_NAME)
+    # only create user_file if it doesn't exist
+    del_flag = False
+    if not os.path.exists(user_file):
+        with open(DEFAULT_PATHS_INI) as infile, open(user_file, "w") as outfile:
+            for line in infile:
+                if CHECK_NAME in line:
+                    line = f"{CHECK_NAME} = {CHANGE_NAME}\n"
+                else:
+                    line = line.replace("/lustre/storeB/project", "${HOME}")
+                outfile.write(line)
+        del_flag = True
+
+    # read default paths.ini
+    # compare
+    assert os.path.exists(user_file)
```

Comment:
Also please remove dead code.

Reply:
done (with the addidtion why some comments were left anyway)